### PR TITLE
rgga urls update

### PR DESCRIPTION
**Pull request by zwiastunsw**
all url's for the RGAA have been changed to the RGAA version on [tanaguru.com](https://rgaa.tanaguru.com/)

### CODE REVIEW

**Review by ffoodd — COMMENTED** [review]
Thanks for the PR ♥️

I think you linked the RGAA v3, however the current version is 4.1. Would you mind to update the links ?

No worry if not, I'll take care of it 🙂

**Comment by zwiastunsw** [comment]
Hi, Yes, I will look into it. There are some tests that don't exist in RGAA 4, so it was simpler for me to update all the links first. Now I need to compare which tests are not there and which have changed their numbering.

**Comment by ffoodd** [comment]
Thanks a lot, this is really helpful!

File: sass/themes/_error.scss [hunk 1]
```diff
@@ -585,7 +584,7 @@ Use `[disabled]` and `[readonly]`.
 
 ### Selector
 ```css
-button[class*="disabled"]:not([disabled], [readonly])
+button[class*="disabled"]:not([disabled]):not([readonly])
```
Comment (zwiastunsw) [inline sass/themes/_error.scss:587]: should be: 
button[class*="disabled"]:not([disabled], [readonly])

**Review by ffoodd — COMMENTED** [review]
Hi there, I'll keep this open until I get some time to check those links. FYI RGAA just got an update and is currently 4.1.1.

**Comment by zwiastunsw** [comment]
Hi, 
These are not all the changes. 
A large group of links to wai-aria do not exist, for example in _error.scss file: 

`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-checked>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-checked>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-expanded>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-expanded>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-controls>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-controls>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-valuemin>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-valuemin>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-valuemax>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-valuemax>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-valuenow>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#aria-valuenow>`
`<https://www.w3.org/TR/wai-aria/states_and_properties#aria-orientation>` new URL in WAI-ARIA 1.2 https://www.w3.org/TR/wai-aria-1.2/#aria-orientation>`

`<https://www.w3.org/TR/wai-aria/roles#img>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#img>`
`<https://www.w3.org/TR/wai-aria/roles#presentation>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#presentation>`

`<https://www.w3.org/TR/wai-aria/roles#textalternativecomputation>` (no exist in WAI-ARIA 1.2

`<https://www.w3.org/TR/wai-aria/roles#slider>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#slider>`
`<https://www.w3.org/TR/wai-aria/roles#spinbutton>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#spinbutton>`
`<https://www.w3.org/TR/wai-aria/roles#checkbox>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#checkbox>`
`<https://www.w3.org/TR/wai-aria/roles#combobox>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#combobox>`
`<https://www.w3.org/TR/wai-aria/roles#scrollbar>` new URL in WAI-ARIA 1.2 `<https://www.w3.org/TR/wai-aria-1.2/#scrollbar>`

Once you approve and merge this PR, I will prepare another PR

File: sass/themes/_error.scss [hunk 2]
```diff
@@ -756,11 +754,11 @@ category: Errors
...
-* <https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-8-3>
-* <https://references.modernisation.gouv.fr/rgaa-accessibilite/glossaire.html#langue-par-dfaut>
-* <https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-8-4>
-* <https://www.w3.org/TR/WCAG20/#meaning-doc-lang-id>
-* <https://www.w3.org/TR/WCAG-TECHS/H57.html>
+* <https://rgaa.tanaguru.com/rgaa4-criteres.html#crit-8-3>
+* <https://rgaa.tanaguru.com/rgaa4-glossaire.html#glossaire-61>
+* <https://rgaa.tanaguru.com/rgaa4-criteres.html#crit-8-4>
+* <https://www.w3.org/TR/WCAG21/#language-of-page>
+* <https://www.w3.org/WAI/WCAG21/Techniques/html/H5>
```
Comment (zwiastunsw) [inline sass/themes/_error.scss:761]: mistake, should be `<https://www.w3.org/WAI/WCAG21/Techniques/html/H5>`

Comment (ffoodd) [inline sass/themes/_error.scss:761]:
```suggestion
* <https://www.w3.org/WAI/WCAG21/Techniques/html/H57>
```

**Review by ffoodd — CHANGES_REQUESTED** [review]

**Pull request merged**